src/Delivery/DynamicEntry.php

```diff
@@ -88,51 +94,73 @@ public function __call($name, $arguments)
             $getId = true;
         }
 
-        if ($fieldConfig !== null && !$fieldConfig->isDisabled()) {
-            if (!isset($this->fields->$fieldName)) {
-                if ($fieldConfig->getType() === 'Array') {
-                    return [];
-                }
+        if ($fieldConfig === null || $fieldConfig->isDisabled()) {
+            trigger_error('Call to undefined method ' . __CLASS__ . '::' . $name . '()', E_USER_ERROR);
```

Comment:
I think this error message is missleading. Shouldn't it be something like `disabled or non existing field found`?